docs/guide/taxii.ipynb

```diff
@@ -71,2326 +71,79 @@
-      "    \"labels\": [\n",
-      "        \"malicious-activity\"\n",
-      "    ],\n",
-      "    \"name\": \"Emerging Threats - Block Rules - Compromised IPs\",\n",
-      "    \"pattern\": \"[ ipv4-addr:value = '212.27.60.27' ]\",\n",
-      "    \"valid_from\": \"2017-10-02T20:40:44Z\",\n",
-      "    \"kill_chain_phases\": [\n",
-      "        {\n",
-      "            \"kill_chain_name\": \"lockheed-martin-cyber-kill-chain\",\n",
-      "            \"phase_name\": \"delivery\"\n",
-      "        }\n",
-      "    ]\n",
-      "}\n",
-      "{\n",
-      "    \"type\": \"indicator\",\n",
-      "    \"id\": \"indicator--da56b536-6ac7-44d5-a036-0db986926016\",\n",
-      "    \"created\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"modified\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"labels\": [\n",
-      "        \"malicious-activity\"\n",
-      "    ],\n",
-      "    \"name\": \"Emerging Threats - Block Rules - Compromised IPs\",\n",
-      "    \"pattern\": \"[ ipv4-addr:value = '213.236.208.178' ]\",\n",
-      "    \"valid_from\": \"2017-10-02T20:40:44Z\",\n",
-      "    \"kill_chain_phases\": [\n",
-      "        {\n",
-      "            \"kill_chain_name\": \"lockheed-martin-cyber-kill-chain\",\n",
-      "            \"phase_name\": \"delivery\"\n",
-      "        }\n",
-      "    ]\n",
-      "}\n",
-      "{\n",
-      "    \"type\": \"indicator\",\n",
-      "    \"id\": \"indicator--95f9c0f4-351b-43c9-81da-c5fdcfe4fa6d\",\n",
-      "    \"created\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"modified\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"labels\": [\n",
-      "        \"malicious-activity\"\n",
-      "    ],\n",
-      "    \"name\": \"Emerging Threats - Block Rules - Compromised IPs\",\n",
-      "    \"pattern\": \"[ ipv4-addr:value = '94.125.182.252' ]\",\n",
-      "    \"valid_from\": \"2017-10-02T20:40:44Z\",\n",
-      "    \"kill_chain_phases\": [\n",
-      "        {\n",
-      "            \"kill_chain_name\": \"lockheed-martin-cyber-kill-chain\",\n",
-      "            \"phase_name\": \"delivery\"\n",
-      "        }\n",
-      "    ]\n",
-      "}\n",
-      "{\n",
-      "    \"type\": \"indicator\",\n",
-      "    \"id\": \"indicator--5007db19-0906-4aec-b18b-e0819b3f13de\",\n",
-      "    \"created\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"modified\": \"2017-10-02T20:40:44.000Z\",\n",
-      "    \"labels\": [\n",
-      "        \"malicious-activity\"\n",
-      "    ],\n",
-      "    \"name\": \"Emerging Threats - Block Rules - Compromised IPs\",\n",
-      "    \"pattern\": \"[ ipv4-addr:value = '208.83.20.130' ]\",\n",
-      "    \"valid_from\": \"2017-10-02T20:40:44Z\",\n",
-      "    \"kill_chain_phases\": [\n",
-      "        {\n",
-      "            \"kill_chain_name\": \"lockheed-martin-cyber-kill-chain\",\n",
-      "            \"phase_name\": \"delivery\"\n",
-      "        }\n",
-      "    ]\n",
-      "}\n",
-      "{\n",
- 
```

Comment:
Not a big deal either way, but maybe change the password to the one used in medallion's readme and test config (`Password0`).

Reply:
That was a local hack to make sure I was using the right instance of medallion. So agree.

Reply:
and noted on the #64 issue